multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mymangacms/MyMangaCMS.kt

```diff
@@ -230,6 +246,12 @@ abstract class MyMangaCMS(
             .attr("style")
             .let { backgroundImageRegex.find(it)?.groups?.get(1)?.value }
     }
+
+    private fun removeGenericWords(name: String): String {
+        return name.split(' ').filterNot { word ->
+            word.lowercase(Locale.US) in removeGenericWords
+        }.joinToString(" ")
+    }
     //endregion
```

Comment:
This is not locale friendly.

```suggestion
    private fun removeGenericWords(name: String): String {
        val locale = Locale.forLanguageTag(lang)
        
        return name.split(' ')
            .filterNot { word -> word.lowercase(locale) in removeGenericWords }
            .joinToString(" ")
    }
    //endregion
```